docs/releases/5.0.1.txt

```diff
@@ -12,3 +12,7 @@ Bugfixes
 * Reallowed, following a regression in Django 5.0, using a foreign key to a
   model with a primary key that is not ``AutoField`` in
   :attr:`.ModelAdmin.list_filter` (:ticket:`35020`).
+
+* Fixed a long standing bug in handling ``RETURNING INTO`` clause that caused a
+  crash when creating a model instance with ``GeneratedField`` and
+  ``output_field`` had backend-specific converters (:ticket:`35024`).
```

Comment:
I'm not entirely understanding this sentence, did you mean:
```suggestion
  ``output_field`` that had backend-specific converters (:ticket:`35024`).
```

In any case, I would suggest a minor rewording like this:
```suggestion
* Fixed a long standing bug in handling the ``RETURNING INTO`` clause that
  caused a crash when creating a model instance including a ``GeneratedField``
  which ``output_field`` had backend-specific converters (:ticket:`35024`).
```

Reply:
`that` sounds redundant to me.